pages/index.js

```diff
@@ -22,6 +22,14 @@ const Index = () => {
         />
       </Head>
       <HomePage />
+      <footer>
+        <p>
+          The contents of this website are deployed from this 
+          <a href="{https://github.com/Real-Dev-Squad/website-members}" target="_blank">
```

Comment:
No css used?

Reply:
Not yet, I first wanted to make sure this changes work and accordingly next step will be to add css.

Reply:
Not good from a PR submission standpoint. Finish adding the CSS please.

Reply:
Ok, I will add the css and re submit the PR